applications/serial_lte_modem/doc/SOCKET_AT_commands.rst

```diff
@@ -1057,11 +1062,15 @@ Syntax
 
 ::
 
-   #XRECVFROM=<timeout>
+   #XRECVFROM=<timeout>[,<flags>]
 
 * The ``<timeout>`` value sets the timeout value in seconds.
   ``0`` means no timeout, and it makes this request become blocking.
 
+* The ``<flags>`` value sets receiving behavior based on BSD socket definition.
+  ``2`` means reading data without removing it from socket input queue.
+  ``64`` means overriding operation to non-blocking.
+
```

Comment:
Why isn't here `waitall` listed? Don't you support `recvfrom()` for TCP?

Reply:
According to BSD socket spec, `msg_waitall` has no effect on datagram.